src/shared/base-filesystem.c

```diff
@@ -205,3 +205,13 @@ int base_filesystem_create(const char *root, uid_t uid, gid_t gid) {
 
         return 0;
 }
+
+int base_filesystem_create(const char *root, uid_t uid, gid_t gid) {
+        _cleanup_close_ int fd = -EBADF;
+
+        fd = open(ASSERT_PTR(root), O_DIRECTORY|O_CLOEXEC);
```

Comment:
\yhis drops `O_RDONLY|O_NONBLOCK|O_NOFOLLOW`, was that intentional? If so, commit message should explain why